tx/textual/valuerenderer/message.go

```diff
@@ -51,7 +62,15 @@ func (mr *messageValueRenderer) Format(ctx context.Context, v protoreflect.Value
 			continue
 		}
 
-		subscreens, err := vr.Format(ctx, v.Message().Get(fd))
+		subscreens := make([]Screen, 0)
+		if fd.IsList() {
```

Comment:
@joeabbey what's your opinion on keeping repeated value rendering in message.go vs in a separate struct in repeated.go?

Reply:
Largely indifferent.  Handling can easily go in either place.  

The `repeated` keyword is applies to a `field` and since Messages are composed of fields... Fields cannot exist on their own... it does make sense to keep repeated handled within Messages.

https://developers.google.com/protocol-buffers/docs/proto3#specifying_field_rules